src/styles/blocks/card.scss

```
@@ -0,0 +1,92 @@
+.card {
+  display: flex;
+  max-width: 200px;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+
+  &__img {
+    display: flex;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+
+  &__title {
+    padding-top: 40px;
```

use margin to make outer indent between elements
```suggestion
    margin-top: 40px;
```

---

ok